a10_octavia/controller/worker/tasks/server_tasks.py

```diff
@@ -147,16 +152,36 @@ def execute(self, member, vthunder, pool, member_count_ip, member_count_ip_port_
             raise e
 
         try:
+            if mem_count_ip_dict and member.ip_address in mem_count_ip_dict:
+                mem_cnt_ip = mem_count_ip_dict[member.ip_address]
+                member_count_ip = mem_cnt_ip
+
+            if (mem_count_ip_port_protocol_dict and
+                    member.ip_address in mem_count_ip_port_protocol_dict):
+                mem_cnt_ip_port_proto = mem_count_ip_port_protocol_dict[member.ip_address]
+                member_count_ip_port_protocol = mem_cnt_ip_port_proto
+
             if member_count_ip <= 1:
```

Comment:
Can test this scenario?
create 2 members:
```
  member {
    address       = "10.0.12.54"
    subnet_id     = "da29e3b4-c885-4834-b4ee-228d7d1bfac1"
    protocol_port = 95
    name          = "m6"
    weight        = 10
  }

  member {
    address       = "10.0.12.54"
    subnet_id     = "da29e3b4-c885-4834-b4ee-228d7d1bfac1"
    protocol_port = 96
    name          = "m7"
    weight        = 15
  }
```
and then delete both of them by batch member update. Will the slb server be delete on thunder? since the member_count_ip should be 2 in this case.